pkg/kubectl/resource/builder.go

```diff
@@ -212,6 +212,9 @@ func (b *Builder) Path(recursive bool, paths ...string) *Builder {
 
 		b.paths = append(b.paths, visitors...)
 	}
+	if len(b.paths) == 0 && len(b.errs) == 0 {
```

Comment:
> Why check if b.errs is empty?

If the path cannot be accessed, b.paths is empty. I think file extension is not the reason, we do not need to return the message.